wfe/web-front-end.go

```diff
@@ -769,6 +769,7 @@ func (wfe *WebFrontEndImpl) challenge(authz core.Authorization, response http.Re
 	switch request.Method {
 	case "GET":
 		challenge := authz.Challenges[challengeIndex]
+		challenge.ValidationRecord = nil
```

Comment:
To be clear: I absolutely think we should expose this to the public. If you're fine with including extra fields that aren't yet in the spec, I'm fine with that. Alternately, let's just document the field in the spec.